maquetado/scss/variables/_variables.scss

```diff
@@ -1,4 +1,4 @@
-$nav-height: 70px;
+$nav-height: 90px;
```

Comment:
why did you changed this, its mandatory to be `70px`

Reply:
I changed it because in the navbar card, they gave me a new size for that variable
![image](https://user-images.githubusercontent.com/21252506/59779616-2ffaa200-927e-11e9-897e-185d010ec626.png)